src/components/autocomplete/Autocomplete.vue

```diff
@@ -664,7 +664,7 @@ export default {
         },
         updateAppendToBody() {
             const dropdownMenu = this.$refs.dropdown
-            const trigger = this.$refs.input.$el
+            const trigger = this.$parent.$data._isTaginput ? this.$parent.$el : this.$refs.input.$el
```

Comment:
How does this fix the positioning problem?

Reply:
The appended to body dropdown is absolutely positioned and its bounds are calculated from `const trigger` element bounds. In standalone Autocomplete, align left dropdown with input makes sense, because input left bound remains fixed. But, when inside Taginput, after adding tags, the input left bound changes, causing the bug (I think this is, at least, a consistency bug, considering left position of not appended-to-body dropdown). So, it tests If it's inside Taginput (test already used inside component) and chooses right trigger accordingly.